guizero/__init__.py

```diff
@@ -1,9 +1,10 @@
 __name__ = "guizero"
 __package__ = "guizero"
-__version__ = '1.5.0'
+__version__ = "1.5.1"
```

Comment:
1.6.0 feels more appropriate? There is some significant change?!

Reply:
Yes, I agree. Not sure what I was doing there, I was rushing to get the PR done before a meeting 😂  I'll change it